client/app/scripts/constants/naming.js

```diff
@@ -1,2 +1,3 @@
 
 export const EDGE_ID_SEPARATOR = '-';
+export const GRAPH_COMPLEXITY_THRESH = 500;
```

Comment:
Changed to move this threshold into the helper, since it will probably only every be used there.